front/admin/cli.ts

```diff
@@ -571,6 +573,39 @@ const productionCheck = async (command: string, args: parseArgs.ParsedArgs) => {
   }
 };
 
+async function apikeys(command: string, args: parseArgs.ParsedArgs) {
+  switch (command) {
+    case "set-role": {
+      if (!args.wId) {
+        throw new Error("Missing --wId argument");
+      }
+
+      if (!args.name) {
+        throw new Error("Missing --name argument");
+      }
+
+      if (!args.role || !isRoleType(args.role)) {
+        throw new Error(
+          "Missing or Incorrect --role argument. Must be admin | user | builder."
+        );
+      }
+
+      const auth = await Authenticator.internalAdminForWorkspace(args.wId);
+
+      const [numAffected] = await KeyResource.updateRole(auth, {
```

Comment:
Sorry final request. You should add a fetchByName(auth, name) and then an updateRole on the object (not static).

That way your code is more reusable and you can check that the key exist before updating it. If the name does not exist here it will silently do nothing.